eng/Dependencies.props

```diff
@@ -83,6 +83,7 @@ and are generated based on the last package release.
     <LatestPackageReference Include="Stashbox.Extensions.Dependencyinjection" Version="2.6.3" />
     <LatestPackageReference Include="StructureMap.Microsoft.DependencyInjection" Version="1.4.0" />
     <LatestPackageReference Include="Unity.Microsoft.DependencyInjection" Version="2.1.3" />
+    <LatestPackageReference Include="Castle.Facilities.AspNetCore" Version="5.0.0" />
```

Comment:
This repository shouldn't depend on ASP.NET Core. Can you remove that from the container implementation? None of the other containers rely on ASP.NET Core here.

Reply:
I'm not the maintainer of this package, but I opened an discussion on that repo page. [https://github.com/castleproject/Windsor/issues/487](url)

I'm interested to hear from them too...